extensions-contrib/statsd-emitter/README.md

```diff
@@ -0,0 +1,31 @@
+## introduction
```

Comment:
please see how docs for other extensions are done and move the content of this readme under the extensions-contrib doc folder with a layout_page header inserted

also please update the extensions.md doc